src/core/svgRenderer.coffee

```diff
@@ -4,7 +4,7 @@ renderers = {}
 
```

Comment:
Please revert the changes to this file. They don't belong in this PR as far as I can tell.